pkg/collectors/pool.go

```diff
@@ -310,41 +440,94 @@ func createLogicalCapacityPoolMetrics(ch chan<- prometheus.Metric, f *PerfCollec
 		return
 	}
 
+	//childPoolCapacity, err := strconv.ParseFloat(pool[ChildPoolCapacityKey].(string), 64)
+	//if err != nil {
+	//	log.Errorf("get Child Pool Capacity failed: %s", err)
+	//	return
+	//}
```

Comment:
are we adding the `childPoolCapacity` field to our UI? 
if not, then I think we can remove those lines.

Reply:
Actually we should according to their instructions ... 
but that creates a discrepancy with their UI as well ...

Reply:
removed